client/src/components/workbook/modals/AddSimModal.tsx

```diff
@@ -23,24 +24,26 @@ const AddSimModal = (props: Props) => {
   };
 
   const getSim = (sim: any) => {
```

Comment:
This method seems to do the opposite of what its name specifies!

Reply:
Yes. Corrected the same.